conf/AutoBalance.conf.dist

```diff
@@ -75,15 +75,17 @@ AutoBalance.levelUseDbValuesWhenExists = 0
 AutoBalance.LevelEndGameBoost = 1
 
 #
-#     AutoBalance.Instances
-#        Set instances to Auto chance XPlayer depending on players in it.
-#        Default:     1 (1 = ON, 0 = OFF)
+#     AutoBalance.DungeonFullGroupXP
+#        Ensure that the player always gets the same XP as if entering the dungeon with a full group
```

Comment:
Ok i tried this:

```suggestion
#        Decrease individual player's amount of XP gained during a dungeon to match the amount of XP gained during a full group run. Ex: In a 5-man group, you earn 1/5 of the total XP of the dungeon, but if you solo the dungeon with AutoBalance.DungeonFullGroupXP = 0, you will earn 5/5 of the total XP. With the option, you will earn 1/5.
```

and for the option name, use one suggested in discord and replace it in the suggestion above :P

Reply:
Done